schemas/json/layout/layout.schema.v1.json

```
@@ -1215,6 +1216,12 @@
         }
       }
     },
+    "alertOnDelete":{
+        "type": "boolean",
```

```suggestion
        "$ref": "expression.schema.v1.json#/definitions/boolean",
```

I think this was supposed to be an expression, and it seems to be one in groups now, at least. To make that work, a configuration should also be added to `ExprConfigForComponent`.

Using an expression for this value can be useful to indicate that some rows in a repeating group should alert on deletion (for example if they have caused any effects on other parts of the data model), but it's probably less useful (although not entirely pointless) for attachments.

---

I see, that is a good point! Thx for the good explanation as well. I added `alertOnDelete` to `ExprConfigForComponent` as you said.